CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Fixes
 
 - Warn users about multiple versions of `promise` package which can cause unexpected behavior like undefined `Promise.allSettled` ([#3162](https://github.com/getsentry/sentry-react-native/pull/3162))
+- `event` isn't enriched with extra context after it leaves `beforeSend` JS callback on Android ([#3170](https://github.com/getsentry/sentry-react-native/pull/3170))
```

Comment:
I'd rather make this message more user-friendly, for example.

> Event is enriched with all the Android context on the JS layer and you can filter/modify all the data on the `beforeSend` callback similarly to iOS.

Something like that, so it's clear the advantage.